Cork/Views/Sidebar/Sidebar View.swift

```diff
@@ -43,7 +43,7 @@ struct SidebarView: View
                                     try await uninstallSelectedPackage(package: formula, brewData: brewData, appState: appState)
                                 }
                             } label: {
-                                Text("Uninstall \(formula.name)")
+                                Text("Uninstall")
```

Comment:
When this right-click menu is being shown, the name of the formula/cask is already being highlighted in the sidebar

Reply:
I wanted to repeat it because I wanted the user to double-check that they're uninstalling what they actually want to uninstall. 

I also don't like that when you want to double-check what you're uninstalling, without it being included in the right-click menu, you'd have to finish reading the "Uninstall" button, then dart your eyes back to the very left to read what you're uninstalling, and then dart them back to the "Uninstall" button, which is too many steps IMO.

I think we should leave this one as it is, even if it's not what default Mac apps do

Reply:
Ok, will revert